src/App.js

```diff
@@ -31,12 +31,13 @@ const App = () => {
       <br />
       <br />
       <Retool
-        url="https://support.retool.com/embedded/public/cb9e15f0-5d7c-43a7-a746-cdec870dde9a"
+        url="https://retoolin.tryretool.com/embedded/public/9e6c7390-922d-41bd-b038-2e9c0476867c"
```

Comment:
There's a bit of functionality (message passing to/from parent) in the other app that I think is good to keep around. Could you add the downloading to the existing sample app? I've invited you to support.retool.com, app is here: https://support.retool.com/editor/56f90c0c-379a-11ec-a25e-43838d4022ed/Ben/Sample%20React%20Retool

Reply:
Sure! Sorry about that, will update